### DISCUSSION TITLE
How to calculate Spectral Efficient or Throughput in the Sionna?

### DISCUSSION THREAD

**Discussion by 571991380** [NVlabs/sionna#439] [Q&A]
Dear Sionna Team,

We can directly calculate the BLER/BER with the Sionna API.
But I'm curious how to calculate Throughput in the Sionna?
Thank you so much.

**Comment by 571991380** [comment]
I am using the ”Realistic_Multiuser_MIMO_Simulations“case。

**Comment by SebastianCa** [comment]
This discussion is redundant with #343.
